app/models/participant_profile/mentor.rb

```diff
@@ -22,9 +22,44 @@ class ParticipantProfile::Mentor < ParticipantProfile::ECF
   }
 
   def self.archivable_from_frozen_cohort(restrict_to_participant_ids: [])
-    super(restrict_to_participant_ids:)
-      .where(mentor_completion_date: nil)
-      .where.not(id: InductionRecord.where.not(mentor_profile_id: nil).select(:mentor_profile_id).distinct)
+    archivable_states = %i[ineligible voided submitted].freeze
```

Comment:
Is it worth moving this to a constant on `ParticipantDeclaration`? we could then test all non-archivable states with something like:

```
(ParticipantDeclaration.states.keys - ParticipantDeclaration::ARCHIVABLE_STATES).each |non_archivable_state| do
  # Test it can't be archived
end
```